couchdb-audit/node.js

```diff
@@ -13,12 +13,19 @@ module.exports = {
    *
    * @name withFelix(felix, name)
    * @param {Object} felix The felix instance to use to persist.
+   * @param {Object} felixAudit Optionally a different felix instance to use to persist audit data
    * @param {String|Function(err, name)} name Either the authors name,
    *    or a function to retrieve the name.
    * @api public
    */
-  withFelix: function(felix, name) {
-    return log.init(felix.name, felix.client, felix, getNameFn(name));
+  // withFelix: function(felix, felixAudit, name) {
+  withFelix: function(felix, felixAudit, name) {
+    if (arguments.length === 2) {
+      name = arguments[1];
```

Comment:
Yeah that's a much better idea. It also makes something clear which may not be: that the order you reassign these actually matters